src/IlkRegistry.sol

```diff
@@ -328,25 +371,62 @@ contract IlkRegistry {
     function update(bytes32 ilk) external {
         require(JoinLike(ilkData[ilk].join).vat() == address(vat), "IlkRegistry/invalid-ilk");
         require(JoinLike(ilkData[ilk].join).live() == 1, "IlkRegistry/ilk-not-live-use-remove-instead");
+        uint96 _class = ilkData[ilk].class;
+        require(_class == 1 || _class == 2, "IlkRegistry/invalid-class");
```

Comment:
So, even though we know RWA is coming soon and we plan to use 3+ for such ilks, we'll just plan to redeloy the registry? The alternative would be some sort of "addClass" privileged functionality that only the pause proxy can call.

Reply:
Ah nvm, I see the function below provides an equivalent ability.